modules/Package.swift

```diff
@@ -75,12 +80,14 @@ let package = Package(
         .library(name: "ZcashSDKEnvironment", targets: ["ZcashSDKEnvironment"])
     ],
     dependencies: [
+//        .package(url: "https://github.com/pacu/zcash-swift-payment-uri", from: "0.1.0-beta.7"),
```

Comment:
@LukasKorba I've transfered the payment URI library to https://github.com/zecdev/zcash-swift-payment-uri